app/models/item.rb

```diff
@@ -0,0 +1,8 @@
+class Item < ActiveRecord::Base
+  belongs_to :user
+  belongs_to :category
+  belongs_to :price_range
```

Comment:
Associations in lines 3 and 4 should not be here yet. Those tables are not set up. By adding them to the model you are creating methods that are broken.